telegraf/rootfs/config.toml.tpl

```diff
@@ -256,6 +257,15 @@
   data_format = {{ default "influx" .NSQ_CONSUMER_DATA_FORMAT | quote }}
 {{ end }}
 
+{{ if .DEIS_CONTROLLER }}
+[[inputs.deis_controller]]
+  server = {{ .DEIS_CONTROLLER | quote }}
+  topic = {{ default "telegraf" .DEIS_CONTROLLER_TOPIC | quote }}
```

Comment:
we should probably make this metrics by default since that is what we use for the router metrics.